native/start.sh

```diff
@@ -0,0 +1,2 @@
+python3 native.py -n 6 -d 5 -l 1 --port 30303 --websocket= --transaction-sender=0 --ds-guard=4 --shard-guard=0 --bucket=zilliqa-devnet  --origin-server=/home/stephen/dev/Zilliqa/rundirs --multiplier-fanout=1 --out-dir=/home/stephen/dev/native/rundirs  --build-dir=/home/stephen/dev/Zilliqa/build --skip-non-guard-ds
```

Comment:
Make this generic as it has a hard coded dir name in it, even though this is just an example it will attract noise.

Reply:
new scripts provided

setup.sh
startall.sh
stopall.sh